industrial_extrinsic_cal/launch/pose_verification.launch

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" ?>
+<launch>
+  <arg name="model" />
+  <param name="robot_description" textfile="$(find abb_irb1600_cal_support)/urdf/irb1600_6_12.urdf"/>
+  <param name="use_gui" value="true"/>
+
+
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+
+  <node pkg="tf" type="static_transform_publisher" name="camera_poses_that_see_target" args=".6 0 -.4 0 0 0 /world /target2 25"/>
+  <node pkg="tf" type="static_transform_publisher" name="all_camera_poses" args=".6 0 -.4 0 0 0 /world /target 25"/>
+  <node pkg="industrial_extrinsic_cal" type="position_validation" name="position_validation" output="screen">
+    <rosparam>
+      camera_file: "test2_camera_def.yaml"
+      target_file: "test2_target_def.yaml"
+      cal_job_file: "test2_caljob_def.yaml"
+    </rosparam>
+    <param name="robot_urdf" type="str" value="$(find abb_irb1600_cal_support)/urdf/irb1600_6_12.urdf"/>
+    <param name="target_yaml" type="str" value="$(find industrial_extrinsic_cal)/yaml/ical_srv_target.yaml"/>
+    <param name="base_link" type="str" value="base_link"/>
+    <param name="tool0" type="str" value="tool0"/>
+    <param name="target" type="str" value="target"/>
+    <param name="world" type="str" value="world"/>
+    <param name="image_width" type="int" value="1080"/>
+    <param name="image_height" type="int" value="1080"/>
+    <param name="focal_length" type="double" value="529.2"/>
+    <param name="numberOfStopsForPhotos" type="double" value="8"/>
+    <param name="poseHeight" type="double" value="1.2"/>
+    <param name="spacing_in_z" type="double" value=".25"/>
+    <param name="angleOfCone" type="double" value="1.047197"/>
```

Comment:
Change angle of cone to be entered in degrees. Few people have an intuitive sense of radians